src/Frontend/Page/User.hs

```diff
@@ -39,31 +39,43 @@ import qualified Text.Digestive.Lucid.Html5 as DF
 data PageUserSettings = PageUserSettings User
   deriving (Eq, Show, Read)
 
-instance Page PageUserSettings
+instance Page PageUserSettings where
+    -- The use settings page always goes to the profile of the current logged in user.
```

Comment:
It hasn't changed yet but with 58d2aa51cf5b5c36af82bf949130af2ab3a4deaa we no longer have this assumption.